rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java

```diff
@@ -208,6 +273,30 @@ JobId submitFromCatalog(String catalogRestURL, String bucketName, String workflo
             Map<String, String> variables)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)